network/tc.go

```diff
@@ -56,20 +56,24 @@ const (
 	tcPriority uint32 = uint32(1000)
 )
 
+//go:generate mockery --name=TrafficController --filename=tc_mock.go
+
 // TrafficController is an interface being able to interact with the host
 // queueing discipline
 type TrafficController interface {
 	AddNetem(ifaces []string, parent string, handle string, delay time.Duration, delayJitter time.Duration, drop int, corrupt int, duplicate int) error
 	AddPrio(ifaces []string, parent string, handle string, bands uint32, priomap [16]uint32) error
-	AddFilter(ifaces []string, parent string, handle string, srcIP, dstIP *net.IPNet, srcPort, dstPort int, protocol Protocol, connState connState, flowid string) (uint32, error)
+	AddFilter(ifaces []string, parent string, handle string, srcIP, dstIP *net.IPNet, srcPort, dstPort int, prot protocol, state connState, flowid string) (uint32, error)
 	DeleteFilter(iface string, priority uint32) error
 	AddFwFilter(ifaces []string, parent string, handle string, flowid string) error
 	AddOutputLimit(ifaces []string, parent string, handle string, bytesPerSec uint) error
 	ClearQdisc(ifaces []string) error
 }
 
+//go:generate mockery --name=tcExecuter --filename=tc_executer_mock.go
+
 type tcExecuter interface {
-	Run(args ...string) (exitCode int, stdout string, stderr error)
```

Comment:
[Variadic](https://vektra.github.io/mockery/notes/#variadic-arguments) arguments does not act as we expect

Using `unroll-variadic: False` make the generation of the mock failing, let's keep it simpler for now